test/cql-pytest/run

```diff
@@ -14,6 +14,23 @@ else:
     cmd = run.run_scylla_cmd
     check_cql = run.check_cql
 
+# If the "--tablets" option is given, switch to the experimental Tablets-
+# based replication instead of vnodes. Some tests are expected to fail
+# when using tablets because of unimplemented features, so they use the
+# fails_with_tablets fixture that will cause them to xfail when tablets
+# are used.
+# TODO: In the near future, we should reverse this option: tests should run
+# with the tablets experimental feature by default, and a "--vnodes"
+# option will ask to run them without the tablets experimental feature.
+if '--tablets' in sys.argv:
+    sys.argv.remove('--tablets')
+    def run_with_tablets(pid, dir):
+        (c, e) = run_with_tablets.orig_cmd(pid, dir)
+        return (c + ['--experimental-features=consistent-topology-changes',
+                     '--experimental-features=tablets'], e)
+    run_with_tablets.orig_cmd = cmd
+    cmd = run_with_tablets
```

Comment:
This is not required. We should unconditionally enable tablets in `run.py` and in `suite.yaml`. This is what is going to happen soon, anyway. This is what I did in my PR.

Reply:
I can do this, but this will mean the test behavior on master will immediately change after my patch, and I didn't want to do this, and I don't think it's urgent. I thought that for now, developers can run tablet tests without them being "the default" in any way.

But if you think it should be the default now, it will be easy. So instead I'll need to do the **reverse** of this patch - enable the tablets experimental feature unless an option (e.g., --vnodes) is given. So this code **is** required, just in reverse.